doc/source/conf.py

```diff
@@ -97,7 +97,11 @@ def fmt(string):
     'sphinx.ext.doctest',
     'sphinx.ext.autosummary',
     'sphinx.ext.intersphinx',
-    'nbsphinx'
+    'nbsphinx',
+    # this extenstion is needed to avoid the
+    # "WARNING: Pygments lexer name 'ipython3' is not known" error for the notebooks.
+    # fix based on: https://github.com/spatialaudio/nbsphinx/issues/24
+    'IPython.sphinxext.ipython_console_highlighting'
```

Comment:
wait... we don't have those warnings: http://biocore.github.io/calour/notebooks/microbiome_step_by_step.html

Reply:
i don't understand :(
if you do (in the docs):
make clean
make html

you get a lot of : "WARNING: Pygments lexer name 'ipython3' is not known" warnings
this fixes these warnings